src/seccomp_fd_notify.c

```diff
@@ -103,22 +115,41 @@ static int install_notify_filter(void) {
 	return notify_fd;
 }
 
+int get_mem_fd(int pid) {
+	char mem_path[6 /* /proc/ */
+		      + 22 /* INTTYPE_TO_STRLEN(int64_t) */
+		      + 3 /* mem */
+		      + 1 /* \0 */];
+	snprintf(mem_path, sizeof(mem_path), "/proc/%d/mem", pid);
```

Comment:
are you sure /proc is mounted (and valid) at this point?

Reply:
I would be surprised if it wasn't? I would be surprised if containerd is spawning processes before the container is fully setup, including its mounts/filesystem.